### PR TITLE
Update firewalld-beginners.md (punctuation, wording, ...)

### PR DESCRIPTION
**Pull request by SergeCroise**
1 punctuation,
2. short guide on the subject
3. single for, ...

#### Author checklist (Completed by original Author)
- [x] Good fit for the Rocky Linux project? Title and Author Metatags inserted ?
- [x] If applicable, steps and instructions have been tested to work
- [x] Initial self-review to fix basic typos and grammar completed

#### Rocky Documentation checklist (Completed by Rocky team) 
- [x] 1st Pass (Document is good fit for project and author checklist completed)
- [ ] 2nd Pass (Technical Review - check for technical correctness) 
- [ ] 3rd Pass (Detailed Editorial Review and Peer Review)
- [ ] Final approval (Final Review)

### CODE REVIEW

**Comment by SergeCroise** [comment]
Hi @gannazhyrnova,
please review.

File: docs/guides/security/firewalld-beginners.md [hunk 1]
```diff
@@ -38,8 +37,8 @@ You'll need:
 
 * A Rocky Linux machine of any kind, local or remote, physical or virtual
 * Access to the terminal, and a willingness to use it
-* You need root access, or at least the ability to use `sudo` on your user account. For simplicity's sake, I'm assuming all commands are being run as root.
-* A basic understanding of SSH wouldn't hurt for managing remote machines.
+* You need root access, or at least the ability to use `sudo` on your user account. For simplicity's sake, I'm assuming all commands are being run as root
+* A basic understanding of SSH wouldn't hurt for managing remote machines
```
Comment (sspencerwire) [inline docs/guides/security/firewalld-beginners.md:41]: Removing the punctuation here is OK. The style guide will let you put a period or an end in in the last item, though, so it isn't technically *wrong* as originally written.

**Comment by sspencerwire** [comment]
@EzequielBruni take a look at some of the suggested changes made by @SergeCroise here. I've also made some recommendations on wording changes to avoid some problematic English language rules on hyphenation. :-)  Let us know what you think. 

@SergeCroise please see the items that need to be fixed that I've noted.  Thank you!

File: docs/guides/security/firewalld-beginners.md [hunk 2]
```diff
@@ -15,14 +14,14 @@ Of course, if you're here, you probably have a better idea what a firewall is an
...
-So let's talk about what we're here for. `firewalld` is the default firewall app packaged with Rocky Linux, and it's designed to be pretty simple to use. You just need to know a little bit about how firewalls work, and not be afraid to use the command line.
+So, let's talk about what we're here for. `firewalld` is the default firewall app packaged with Rocky Linux, and it's designed to be pretty simple to use. You just need to know a little bit about how firewalls work, and not be afraid to use the command line.
 
 Here you'll learn:
 
 * The very basics of how `firewalld` works
 * How to use `firewalld` to restrict or allow incoming and outgoing connections
 * How to allow only people from certain IP addresses or places to log into your machine remotely
-* How to manage some `firewalld`-specific features like Zones.
+* How to manage some `firewalld`-specific features like Zones .
```
Comment (EzequielBruni) [inline docs/guides/security/firewalld-beginners.md:24]: Why did you add a space between "Zones" and the period? Did you mean to remove the period altogether? I'm fine with that.

File: docs/guides/security/firewalld-beginners.md [hunk 3]
```diff
@@ -138,14 +137,14 @@ firewall-cmd --permanent [the rest of your command]
 
 Before anything else, I need to explain zones. Zones are a feature that basically allow you to define different sets of rules for different situations. Zones are a huge part of `firewalld` so it pays to understand how they work.
 
-If your machine has multiple ways to connect to different networks (eg. Ethernet and WiFi), you can decide that one connection is more trusted than the other. You might set your Ethernet connection to the "trusted" zone if it's only connected to a local network that you built, and put the WiFi (which might be connected to the internet) in the "public" zone with more stringent restrictions.
+If your machine has multiple ways to connect to different networks (e.g., Ethernet and Wi-Fi), you can decide that one connection is more trusted than the other. You might set your Ethernet connection to the "trusted" zone if it's only connected to a local network that you built , and put the Wi-Fi (which might be connected to the internet) in the "public" zone with more stringent restrictions.
```
Comment (EzequielBruni) [inline docs/guides/security/firewalld-beginners.md:140]: Should probably remove the space between "built" and the comma. There's at least one more instance of an extra space below.

File: docs/guides/security/firewalld-beginners.md [hunk 4]
```diff
@@ -167,7 +166,7 @@ Default zones include the following (I've taken this explanation from [DigitalOc
 
 > **trusted:** Trust all of the machines in the network. The most open of the available options and should be used sparingly.
 
-Okay, so some of those explanations get complicated, but Honestly? The average beginner can get by with understanding "trusted", "home", and "public", and when to use which.
+Okay, so some of those explanations get complicated, but honestly? The average beginner can get by with understanding "trusted", "home", and "public", and when to use which.
```
Comment (EzequielBruni) [inline docs/guides/security/firewalld-beginners.md:170]: Oof. Good catch.

File: docs/guides/security/firewalld-beginners.md [hunk 5]
```diff
@@ -157,7 +156,7 @@ Default zones include the following (I've taken this explanation from [DigitalOc
 
 > **external:** External networks in the event that you are using the firewall as your gateway. It is configured for NAT masquerading so that your internal network remains private but reachable.
 
-> **internal:** The other side of the external zone, used for the internal portion of a gateway. The computers are fairly trustworthy and some additional services are available.
+> **internal:** The other side of the external zone, used for the internal portion of a gateway. The computers are fairly trustworthy , and some additional services are available.
```
Comment (EzequielBruni) [inline docs/guides/security/firewalld-beginners.md:159]: Here it is!

**Comment by EzequielBruni** [comment]
Aside from the other comments I left (the only problem was basically some spaces where spaces shouldn't be), I'm fine with all of these edits.

**Comment by sspencerwire** [comment]
@SergeCroise will you at least fix the space issues noted here?  If you don't want to modify the rest and since @EzequielBruni has no objections, I'll merge the remaining as is.

**Review by sspencerwire — APPROVED** [review]
